src/app/pages/applications/forms/kubernetes-settings.component.ts

```diff
@@ -106,24 +101,14 @@ export class KubernetesSettingsComponent {
       }
     })
     const v4_interface_control = _.find(this.fieldSets[1].config, {'name' : 'route_v4_interface'});
-    const v6_interface_control = _.find(this.fieldSets[1].config, {'name' : 'route_v6_interface'});
     this.appService.getInterfaces().subscribe(interfaces => {
       interfaces.forEach(i => {
         v4_interface_control.options.push({ label: i.name, value: i.name });
-        v6_interface_control.options.push({ label: i.name, value: i.name });
       })
     })
-
   }
 
   beforeSubmit(data) {
-    if (data.route_v4_gateway === '') {
-      data.route_v4_gateway = null;
-      console.log('yo', data.route_v4_gateway)
-    }
-    if (data.route_v6_gateway === '') {
-      data.route_v6_gateway = null;
-    }
+    data.node_ip = '0.0.0.0';
```

Comment:
Not sure why we made this changed? Someone requested?

Reply:
yes it was, but reverted.